Sources/UIGestureRecognizer.swift

```diff
@@ -78,24 +85,18 @@ private extension UIGestureRecognizer {
     private func cancelOtherGestureRecognizersThatShouldNotRecognizeSimultaneously() {
         guard let touch = self.trackedTouch else { return }
 
-        let otherRecognizersThatHaveBeganToRecogize = touch.gestureRecognizers.filter {
-            $0 != self && $0.state == .began
+        let otherRecognizers = touch.gestureRecognizers.filter {
+            $0 != self && ($0.state == .began || $0.state == .possible)
```

Comment:
not sure if this internally matches iOS behaviour, but it was needed to achieve the expected UX behaviour. We do now cancel any other recognizer proactively. In our case this means we cancel the sheet pan recognizer when dragging the loop handle

Reply:
we should cancel the pan recognizer as soon as the loop handle recogniser turns to _`.changed`_, i.e. when it starts actually doing something, not as soon as we click the drag handle (which is what `.began` means – I think!)

Reply:
I think you are right, and as I said in the comment above this is what we already implemented for the view lifecycles. But it makes sense to also bring this into the gesture logic

I think something like this is correct behaviour
```swift
if 
    $0 === UItapGestureRecognizer && $0.state == .began ||
    $0 === UIPanGestureRecognizer && $0.state == .moved {
        cancel()
}
```

Reply:
because AFAIK tap gestures do not "move" at all. I could be wrong though

Reply:
AFAIK there is no such state as `.moved`? Only `.changed`, which is generic for all gesture recognizers. Would it be possible to use `.changed` for all types of recognizers but also ensure the `.changed` state is used for the TapGestureRecognizer? Or does that just not make sense?